metrics_activities_exports_test.go

```diff
@@ -0,0 +1,111 @@
+package chartmogul
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/davecgh/go-spew/spew"
+)
+
+// Tests creation on an activity export.
+func TestCreateActivitiesExport(t *testing.T) {
+	server := httptest.NewServer(
+		http.HandlerFunc(
+			func(w http.ResponseWriter, r *http.Request) {
+				if r.Method != "POST" {
+					t.Errorf("Unexpected method %v", r.Method)
+				}
+				if r.RequestURI != "/v/activities_export" {
+					t.Errorf("Unexpected URI %v", r.RequestURI)
+				}
+				w.WriteHeader(http.StatusOK)
+				//nolint
+				w.Write([]byte(`{
+							      "id": "7f554dba-4a41-4cb2-9790-2045e4c3a5b1",
+							      "status": "pending",
+							      "file_url": null,
+							      "params": {
+							        "kind": "activities",
+							        "params": {
+							          "activity_type": "contraction",
+							          "start_date": "2020-01-01",
+							          "end_date": "2020-12-31"
+							        }
+							      },
+							      "expires_at": null,
+							      "created_at": "2021-07-12T14:46:56+00:00"
+							    }`))
+			}))
+	defer server.Close()
+	SetURL(server.URL + "/v/%v")
+
+	var tested IApi = &API{
+		AccountToken: "token",
+		AccessKey:    "key",
+	}
+	activities_export, err := tested.MetricsCreateActivitiesExport(&NewMetricsActivitiesExport{
```

Comment:
don't use underscores in Go names; var activities_export should be activitiesExport